### ISSUE TITLE
Not working in Git COMMIT_EDITMSG

### ISSUE THREAD

**Issue by nrser** [streetsidesoftware/vscode-spell-checker#346]
## TL;DR ##

Extensions isn't working in the `COMMIT_EDITMSG` document that pops up when you have VSCode set as your Git editor and run `git commit`.

I've got `git-commit` listed in `cSpell.enabledLanguageIds` array, and have snippets working in it so I don't think I got the language ID wrong.

Any ideas?

---

## Details ##

It seems pretty well established that VSCode users want am editing experience in the built-in Git commit input area better on par with the rest of the app, including spell checking support (#109).

In the meantime, I'm trying out the old-school `git commit` with VSCode setup as my `$EDITOR` so it opens a full document, like so:

<img width="627" alt="Screenshot 2019-05-25 05 13 40" src="https://user-images.githubusercontent.com/207603/58357496-9eca2280-7ead-11e9-92ee-b65760ed6cd1.png">

One of the features I was hopping to achieve most is spell checking (extension's been great BTW, thanks). However, as you can see from the screen shot, the extension does not appear to be picking it up.

My `settings.json` includes:

```
"cSpell.enabledLanguageIds": [
    "asciidoc",
    "c",
    "cpp",
    "crystal",
    "csharp",
    "css",
    "feature",
    "git-commit", // Don't seem to work..?
    "go",
    "handlebars",
    "html",
    "ignore",
    "jade",
    "javascript",
    "javascriptreact",
    "json",
    "jsonc",
    "latex",
    "less",
    "markdown",
    "php",
    "plaintext",
    "pub",
    "python",
    "restructuredtext",
    "ruby",
    "rust",
    "scss",
    "text",
    "typescript",
    "typescriptreact",
    "yaml",
    "yml"
]
```

The `git-commit` language ID has worked fine for snippets, which makes me feel like it's correct.

These commit message files live at `<PROJECT>/.git/COMMIT_EDITMSG`, which is *ignored* in the project as part of the `.git` directory, so maybe that has something to do with it?

I also see from #109 that there are path-based exclude filters on the spell check server, but neither any of the patterns listed there or where they seems to have moved in the current source

https://github.com/streetsidesoftware/vscode-spell-checker/blob/0c2633d1bbd32a636688f14c21ef70998e07101a/packages/_server/src/documentSettings.ts#L42

look like they would apply to the commit message files.

Solutions or suggestions much appreciated, thanks. If a change is needed I can see if I can find some time to take a swing, but time honestly is my most scarce resource at the moment.

**Comment by Jason3S** [comment]
It is possible to enable the spell checker:
In VS Code settings:
``` jsonc
    "cSpell.ignorePaths": [
        "**/node_modules/**",
        "**/package-lock.json",
        "**/vscode-extension/**",
        "**/.git/**",   // <-- Remove this line
        ".vscode"
    ],

```

**Comment by vegerot** [comment]
It looks like the comment (https://github.com/streetsidesoftware/vscode-spell-checker/issues/346#issuecomment-496016524)  by @Jason3S should no longer be necessary as of f940193

**Comment by vegerot** [comment]
However, the issue still persists for me

**Comment by wijon** [comment]
@vegerot I had the same issue. I solved it by setting the option `cSpell.spellCheckOnlyWorkspaceFiles` to `false`. Now spell checking is working inside the `COMMIT_EDITMSG` document.
